docs/topics/native/native-objc-interop.md

```diff
@@ -136,67 +147,70 @@ package my.library
 fun foo() {}
 ```
 
-can be called from Swift like
+Can be called from Swift like this:
 
 ```swift
 MyLibraryUtilsKt.foo()
 ```
 
+See a collection of examples on accessing top-level Kotlin declarations in the Kotlin-Swift interopedia:
+
+* [Top-level functions](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/overview/Top-level%20functions.md)
+* [Top-level read-only properties](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/functionsandproperties/Top-level%20val%20properties.md)
+* [Top-level mutable properties](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/functionsandproperties/Top-level%20mutable%20var%20properties.md)
+
 ### Method names translation
 
-Generally, Swift argument labels and Objective-C selector pieces are mapped to Kotlin
-parameter names. These two concepts have different semantics, so sometimes
-Swift/Objective-C methods can be imported with a clashing Kotlin signature. In this case,
-the clashing methods can be called from Kotlin using named arguments, e.g.:
+Generally, Swift argument labels and Objective-C selector pieces are mapped to Kotlin parameter names. These two concepts
+have different semantics, so sometimes Swift/Objective-C methods can be imported with a clashing Kotlin signature.
+In this case, the clashing methods can be called from Kotlin using named arguments, for example:
 
 ```swift
 [player moveTo:LEFT byMeters:17]
 [player moveTo:UP byInches:42]
 ```
 
-In Kotlin, it would be:
+In Kotlin, it's:
 
 ```kotlin
 player.moveTo(LEFT, byMeters = 17)
 player.moveTo(UP, byInches = 42)
 ```
 
-The methods of `kotlin.Any` (`equals()`, `hashCode()` and `toString()`) are mapped 
-to the methods `isEquals:`, `hash` and `description` in Objective-C, and to the method
-`isEquals(_:)` and the properties `hash`, `description` in Swift.
+Here's how  some methods of `kotlin.Any` are mapped to Swift/Objective-C:
```

Comment:
```suggestion
Here's how functions of `kotlin.Any` are mapped to Swift/Objective-C:
```

As far as I can tell, these are all of the [functions](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-any/#functions) available in `kotlin.Any`. I thought to check because I kept reading the sentence as implying ”...and other functions may be mapped differently.”